content/ledger-objects/check.md

```diff
@@ -0,0 +1,54 @@
+  "Sequence": 2,
+  "index": "49647F0D748DC3FE26BDACBC57F251AADEFFF391403EC9BF87C97F67E9977FB0"
+}
+```
+
+A `Check` object has the following fields:
+
+| Field               | JSON Type        | [Internal Type][] | Description     |
+|:--------------------|:-----------------|:------------------|:----------------|
+| `LedgerEntryType`   | String           | UInt16            | The value `0x0043`, mapped to the string `Check`, indicates that this object is a Check object. |
+| `Account`           | String           | Account           | The sender of the Check. Cashing the Check debits this address's balance. |
+| `Destination`       | String           | Account           | The intended recipient of the Check. Only this address can cash the Check, using a [CheckCash transaction][]. |
+| `Flags`             | Number           | UInt32            | No flags are defined for Checks, so this value is always `0`. |
+| `SendMax`           | String or Object | Amount            | The maximum amount of currency this Check can debit the sender. If the Check is successfully cashed, the destination is credited in the same currency for up to this amount. |
+| `Sequence`          | Number           | UInt32            | The sequence number of the [CheckCreate transaction][] that created this check. |
+| `OwnerNode`         | String           | UInt64            | A hint indicating which page of the sender's owner directory links to this object, in case the directory consists of multiple pages. **Note:** The object does not contain a direct link to the owner directory containing it, since that value can be derived from the `Account`. |
+| `DestinationNode`   | String           | UInt64            | _(Optional)_ A hint indicating which page of the destination's owner directory links to this object, in case the directory consists of multiple pages. |
+| `Expiration`        | Number           | UInt32            | (Optional) Indicates the time after which this Check is considered expired. See [Specifying Time](reference-rippled.html#specifying-time) for details. |
+| `InvoiceID`         | String           | Hash256           | _(Optional)_ Arbitrary 256-bit hash provided by the sender as a specific reason or identifier for this Check. |
+| `SourceTag`         | Number           | UInt32            | _(Optional)_ An arbitrary tag to further specify the source for this Check, such as a hosted recipient at the sender's address. |
+| `DestinationTag`    | Number           | UInt32            | _(Optional)_ An arbitrary tag to further specify the destination for this Check, such as a hosted recipient at the destination address. |
+| `PreviousTxnID`     | String           | Hash256           | The identifying hash of the transaction that most recently modified this object. |
+| `PreviousTxnLgrSeq` | Number           | UInt32            | The [index of the ledger](#ledger-index) that contains the transaction that most recently modified this object. |
+
```

Comment:
The fields in the table are presented in the following order, which I think is more useful (but please let me know if this seems wrong to you):

- The `LedgerEntryType` comes first because that defines what type this object is
- The required fields are all listed before the optional fields, except the "PreviousTxnID" and "PreviousTxnLgrSeq", which... are just out of order. Sorry, I should fix that.